encoders/build.gradle

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2017 SnappyData, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+apply plugin: 'scala'
+
+compileScala.options.encoding = 'UTF-8'
+// fix scala+java mix to all use compileScala which uses correct dependency order
+sourceSets.main.scala.srcDir 'src/main/java'
+sourceSets.test.scala.srcDirs = ['src/test/java', 'src/test/scala',
+                                 'src/dunit/java', 'src/dunit/scala']
+sourceSets.main.java.srcDirs = []
+sourceSets.test.java.srcDirs = []
+
+dependencies {
+
+  compileOnly 'org.scala-lang:scala-library:' + scalaVersion
+  compileOnly 'org.scala-lang:scala-reflect:' + scalaVersion
+
+  compileOnly("org.apache.spark:spark-hive_${scalaBinaryVersion}:${sparkVersion}")
+  testCompile("org.apache.spark:spark-hive_${scalaBinaryVersion}:${sparkVersion}")
+
+  //compile group: 'io.snappydata', name: 'snappydata-store-core', version: snappyStoreVersion
+  if (new File(rootDir, 'store/build.gradle').exists()) {
+    compile project(':snappy-store:snappydata-store-core')
+  } else {
+    compile group: 'io.snappydata', name: 'snappydata-store-core', version: snappyStoreVersion
+  }
+
+  compile project(":snappy-jdbc_${scalaBinaryVersion}")
+
+  compile("org.parboiled:parboiled_${scalaBinaryVersion}:2.1.4") {
+    exclude(group: 'org.scala-lang', module: 'scala-library')
+    exclude(group: 'org.scala-lang', module: 'scala-reflect')
+    exclude(group: 'org.scala-lang', module: 'scala-compiler')
+  }
+
+  compile 'org.apache.tomcat:tomcat-jdbc:8.5.23'
+  compile group: 'com.zaxxer', name: 'HikariCP', version: '2.7.2'
```

Comment:
use tomcatJdbcVersion and hikariCPVersion defined in top-level build.gradle

Reply:
Update tomcat and hikari version with upstream. 
 - tomcatJdbcVersion = '8.5.37'
 - hikariCPVersion = '2.7.9'